library/core/talaiot/build.gradle.kts

```diff
@@ -14,7 +14,7 @@ dependencies {
     api(project(":library:core:talaiot-logger"))
     api(project(":library:core:talaiot-request"))
     implementation("com.github.oshi:oshi-core:5.5.0")
-    implementation("com.gradle:gradle-enterprise-gradle-plugin:3.5.2")
+    api("com.gradle:gradle-enterprise-gradle-plugin:3.5.2")
```

Comment:
Probably worth mentioning that we depend on this version. If the use will upgrade to later one - there are no contracts defined and our implementation might not work

Reply:
good catch, thanks. Reverted